app/assets/javascripts/classes/GraphVM.ts

```diff
@@ -106,151 +108,140 @@ class GraphVM implements StatisticView {
                 this.updateColor(cvm);
             });
             this.data.push(series);
+            this.buffer.push([]);
         }
 
-
         if (this.data[index].data.length == 0 || this.data[index].data[this.data[index].data.length - 1][0] < timestamp) {
-            this.data[index].data.push([timestamp, dataValue]);
+            if (this.paused) {
+                this.buffer[index].push([timestamp, dataValue]);
+            } else {
+                if (this.buffer[index].length > 0) {
+                    this.data[index].data = this.data[index].data.concat(this.buffer[index]);
+                    this.buffer[index] = [];
+                }
+
+                this.data[index].data.push([timestamp, dataValue]);
+            }
         }
     }
 
-    start() {
-        if (this.started == true) {
+    render() {
+        if (this.stop || this.container.clientWidth == 0) {
             return;
         }
-        this.started = true;
-        this.container = document.getElementById(this.id);
 
-        var previousTimestamp: number = 0.0;
-        var animate = (timestamp: number) => {
+        //set min and max
+        var graphMin = 1000000000;
+        var graphMax = -1000000000;
 
-            if (this.stop) {
-                return;
-            }
+        var now = new Date().getTime() - 1000;
+        if (this.paused) {
+            now = this.pauseTime
+        } else {
+            this.pauseTime = now;
```

Comment:
You should add a pause and unpause methods to this class and modify the pause boolean and pauseTime in that method.